docs/setup/vbmh-setup.md

```diff
@@ -0,0 +1,203 @@
+
+### Pre-requisites
+A fully functioning Metal<sup>3</sup> deployment
+
+## Deploying Sushy-Tools and the Virtual Machines
+
+Define and start a default storage pool
+```
+sudo virsh pool-define-as default dir - - - - "/default"
+sudo virsh pool-build default
+sudo virsh pool-start default
+sudo virsh pool-autostart default
+```
+- This is for Sushy-Tools initialization.
+
+Create storage for virtual machines
+```
+sudo qemu-img create -f qcow2 /var/lib/libvirt/images/node1.qcow2 30G
+sudo qemu-img create -f qcow2 /var/lib/libvirt/images/node2.qcow2 30G
+```
+- This is so that one virtual machine acts as a control plane and another as a worker node. You can add as many worker nodes as you wish.
+
+Install Dependencies
+```
+sudo apt install apache2-utils -y
+sudo pip install sushy-tools
+sudo DEBIAN_FRONTEND=noninteractive apt install podman -y
+```
+- apache2-utils is so that we can use `htpasswd`, podman is so that we can run Sushy-Tools in a container
+
+Create Virtual Machines using Virt
+```
+virt-install --name node-1 --memory 4096 --vcpus 2 --disk /var/lib/libvirt/images/node1.qcow2 --network bridge=m3-prov,model=virtio --osinfo detect=on --console pty,target_type=virtio --noautoconsole --graphics vnc --boot nvram.template=/usr/share/OVMF/OVMF_VARS.fd --boot loader=/usr/share/OVMF/OVMF_CODE.secboot.fd --boot loader.secure=no --boot loader.type=pflash --boot loader.readonly=yes --debug -v --machine pc-q35-5.1
+
+virt-install --name node-2 --memory 4096 --vcpus 2 --disk /var/lib/libvirt/images/node2.qcow2 --network bridge=m3-prov,model=virtio --osinfo detect=on --console pty,target_type=virtio --noautoconsole --graphics vnc --boot nvram.template=/usr/share/OVMF/OVMF_VARS.fd --boot loader=/usr/share/OVMF/OVMF_CODE.secboot.fd --boot loader.secure=no --boot loader.type=pflash --boot loader.readonly=yes --debug -v --machine pc-q35-5.1
+
+```
+- This assumes you will be running 1 control plane and 1 worker node.
+
+Create a directory to store Sushy-tools related files
+```
+mkdir ~/vbmc
+```
+
+Create the Sushy Config file
+```
+cat << EOF > ~/vbmc/sushy.config
+SUSHY_EMULATOR_AUTH_FILE = '/root/vbmc/auth.conf'
+SUSHY_EMULATOR_SSL_CERT = '/root/vbmc/cert.pem'
+SUSHY_EMULATOR_SSL_KEY = '/root/vbmc/key.pem'
+SUSHY_EMULATOR_LISTEN_IP = '0.0.0.0'
+SUSHY_EMULATOR_VMEDIA_DEVICES = {
+    "Cd": {
+        "Name": "Virtual CD",
+        "MediaTypes": [
+            "CD",
+            "DVD"
+        ]
+    },
+    "Floppy": {
+        "Name": "Virtual Removable Media",
+        "MediaTypes": [
+            "Floppy",
+            "USBStick"
+        ]
+    }
+}
+EOF
+```
+
+Add this line to /etc/hosts
+```
+192.168.124.99 boot.ironic.suse.baremetal api.ironic.suse.baremetal inspector.ironic.suse.baremetal media.suse.baremetal
+```
+- This is necessary for DNS resolutions for Metal3 in the metal3-demo environment.
+
+Create Password Configuration within `~/vbmc`
```

Comment:
Could we please include a `cd vbmc` step in order not to miss that the commands below should be executed from the same directory?